parm/config/gefs/config.fetch

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
+
+########## config.fetch ##########
+
+echo "BEGIN: config.fetch"
+
+# Get task specific resources
+source "${EXPDIR}/config.resources" fetch
+
+export ICSDIR="@ICSDIR@" # User provided ICSDIR; blank if not provided
+export BASE_IC="@BASE_IC@" # Platform home for staged ICs
+
+export FETCH_YAML_TMPL="${PARMgfs}/stage/master_gefs.yaml.j2"
+export fetch_yaml="stage_atm_cold.yaml"
+
+# Set ICSDIR (if not defined)
+
+if [[ -z "${ICSDIR}" ]] ; then
+
+  ic_ver="20240610"
+
+  if (( NMEM_ENS > 0 )) ; then
+    ensic="${CASE_ENS}"   
+  fi
+
+  if [[ "${DO_OCN:-NO}" == "YES" ]] ; then
+    ocnic="mx${OCNRES}"
+  fi
+
+  export ICSDIR="${BASE_IC}/${CASE}${ensic:-}${ocnic:-}/${ic_ver}"
+
+fi
+
+# Use of perturbations files for ensembles
+export USE_OCN_ENS_PERTURB_FILES="NO"
+export USE_ATM_ENS_PERTURB_FILES="NO"
```

Comment:
I don't think you need this.  These are all `stage_ic` variables.

```suggestion
```

Reply:
Ok, deleted those files